.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [Optional] Please in all future projects, make sure your branch name is descriptive (e.g.: refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. This will make your work more [professional](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches).

- [Optional} Kind;y make sure that all future commit messages are written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). An example of a good commit message is `Add homepage marup to index.html`, while an example of a bad commit is `added html`.

Reply:
- [ ] [OPTIONAL] Please try to implement these suggestions, good ReadMe's can distinct your documentation skills as a developer, making you unique to recruiters or clients. Also, about writing a descriptive Pull request message. How about editing yours to the example he gave?